File: alpaul_gls/Content/Downloadable/PHAO2018092520181018171302/Business/Estado.cs
using System.Collections.Generic;

namespace Business
{
    public class Estado
    {
	    private static Data.Estado obj = new Data.Estado();

        public static int Add(Models.Estado value)
        {
            return obj.Add(value);
        }

        public static int Edit(Models.Estado value)
        {
            return obj.Edit(value);
        }

        public static int Delete(stirng value)
        {
            return obj.Delete(value);
        }

        public static Models.Estado GetById(string ID)
        {
            return obj.GetById(ID);
        }

		public static Models.Estado GetByGUID(string ID)
        {
            return obj.GetByGUID(ID);
        }

        public static List<Models.Estado> GetAll()
        {
            return obj.GetAll();
        }
    }
}using System.Collections.Generic;

namespace Business
{
    public class Estado
    {
	    private static Data.Estado obj = new Data.Estado();

        public static int Add(Models.Estado value)
        {
            return obj.Add(value);
        }

        public static int Edit(Models.Estado value)
        {
            return obj.Edit(value);
        }

        public static int Delete(stirng value)
        {
            return obj.Delete(value);
        }

        public static Models.Estado GetById(string ID)
        {
            return obj.GetById(ID);
        }

		public static Models.Estado GetByGUID(string ID)
        {
            return obj.GetByGUID(ID);
        }

        public static List<Models.Estado> GetAll()
        {
            return obj.GetAll();
        }
    }
}

File: alpaul_gls/Content/Downloadable/PHAO2018092520181018161350/Business/Estado.vb
Imports System.Collections.Generic

Public Class Estado
    Public Shared Function Add(value As Models.Estado) As Integer
        Return Data.Estado.Add(value)
    End Function

    Public Shared Function Edit(value As Models.Estado) As Integer
        Return Data.Estado.Edit(value)
    End Function

    Public Shared Function Delete(value As String) As Integer
        Return Data.Estado.Delete(value)
    End Function

    Public Shared Function GetById(value As String) As Models.Estado
        Return Data.Estado.GetById(value)
    End Function

    Public Shared Function GetByGUID(value As String) As Models.Estado
        Return Data.Estado.GetByGUID(value)
    End Function

    Public Shared Function GetAll() As List(Of Models.Estado)
        Return Data.Estado.GetAll()
    End Function
End Class